src/test/java/SetTest.java

```diff
@@ -32,15 +32,14 @@ public void size() {
     @ValueSource(ints = {1,2,3})
     @DisplayName("Set의 값이 존재하는지 확인")
     public void isContains(int number) {
-        assertTrue(numbers.contains(number));
+        assertThat(numbers.contains(number)).isTrue();
     }
 
     @ParameterizedTest
-    @CsvSource(value = {"1","2","3","4","5"})
+    @CsvSource(value = {"1:true","2:true","3:true","4:false","5:false"}, delimiter = ':')
     @DisplayName("CsvSource를 활용한 ture/false 리턴 테스트")
-    public void isContains_ShouldReturnTrueOrFalseIntegerValues(int number) {
-        Boolean isTrue = numbers.contains(number);
-        assertEquals(true, isTrue);
+    public void isContains_ShouldReturnTrueOrFalseIntegerValues(int number, Boolean isTrue) {
+        assertEquals(isTrue, numbers.contains(number));
```

Comment:
```suggestion
    public void isContains_ShouldReturnTrueOrFalseIntegerValues(int number, boolean expected) {
        assertThat(numbers.contains(number)).isEqualTo(expected);
    }
```
저는 개인적으로 이렇게 사용하는걸 좋아합니다.
parameter의 변수명과 사용법이 명확해 보이거든요.😄